core/src/main/java/jeeves/server/dispatchers/ServiceManager.java

```diff
@@ -1117,10 +1123,15 @@ else if (outPage.isBLOB()) {
 
                                 if (outPage.getContentType() != null
                                     && outPage.getContentType().startsWith("text/plain")) {
```

Comment:
Seems odd to use text/plain when the extension is *.csv.. I think it should be text/csv